src/Microsoft.PowerShell.Commands.Utility/commands/utility/trace/TraceExpressionCommand.cs

```diff
@@ -391,15 +386,15 @@ public override int MaxCapacity
         }
 
         /// <summary>
-        /// Close the stream
+        /// Close the stream.
         /// </summary>
         /// <remarks>
-        /// Causes subsequent calls to IsOpen to return false and calls to
+        /// Causes subsequent calls to IsOpen to return false and calls to.
```

Comment:
Remove added period' it breaks the sentence.

Reply:
Fixed.